python/lsst/ts/standardscripts/data/scripts/system_wide_shutdown.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# This file is part of ts_standardscripts.
+#
+# Developed for the LSST Telescope and Site Systems.
```

Comment:
Please update your template, based on which you create these files, and refer in it to Vera Rubin Observatory. While you're at it, update this and any other file with this old header as well.